Samples/ObservableCounter/Sources/CounterListView.swift

```diff
@@ -0,0 +1,23 @@
+import SwiftUI
+import WorkflowSwiftUI
+
+struct CounterListView: View {
+    @Perception.Bindable
+    var store: Store<CounterListModel>
+
+    var body: some View {
+        let _ = print("CounterListView.body")
```

Comment:
The printing is intentionally left in these samples to show the effect of state changes on body evaluations.